src/user-defined-types/enums.md

```diff
@@ -31,10 +31,10 @@ fn main() {
 
 Key Points:
 
-* Enumerations allow you to collect a set of values under one type
-* Direction has two variants, `Left` and `Right`. These are referred to with the `Direction::..` namespace.
-* PlayerMove shows the three types of variants. Rust will also store a discriminant so that it can determine at runtime which variant is in a value.
-* This might be a good time to compare Structs and Enums:
+* Enumerations allow you to collect a set of values under one type.
+* `Direction` is a type with variants. There are two values of `Direction`: `Direction::Left` and `Direction::Right`.
+* `PlayerMove` is a type with three variants. In addition to the payloads, Rust will store a discriminant so that it known at runtime which variant is in a `PlayerMove` value.
```

Comment:
```suggestion
* `PlayerMove` is a type with three variants. In addition to the payloads, Rust will store a discriminant so that it knows at runtime which variant is in a `PlayerMove` value.
```